firmware/qemu/src/bin/log.rs

```diff
@@ -482,6 +482,8 @@ fn main() -> ! {
         defmt::info!("{:?}", Display2Format::<consts::U32>(&addr));
     }
 
+    defmt::info!("Hello {:char}", '💜');
```

Comment:
could you also test formatting a `char` value using `{:?}`?

Reply:
Yes, good idea. :+1: